bin/check-ssl-qualys.rb

```diff
@@ -107,11 +129,37 @@ def ssl_check(from_cache)
 
   def ssl_recheck
     1.upto(config[:num_checks]) do |step|
-      json = ssl_check(step != 1)
+      p "step: #{step}" if config[:debug]
+      start_time = Time.now
+      p "start_time: #{start_time}" if config[:debug]
+      json = if step == 1
+               ssl_check(false)
+             else
+               ssl_check(true)
+             end
       return json if json['status'] == 'READY'
-      sleep(config[:between_checks])
+      if json['endpoints'] && json['endpoints'].is_a?(Array)
```

Comment:
Ruby 2.2 is supposed to be EOL'd in 2 weeks.  It sure would be nice to be able to use the safe nav operator here...

Reply:
I feel you, it's on my list of things to run through all the plugins after it is EOL and remove support for ruby `< 2.3`